extensions/sql-bindings/src/test/common/azureFunctionsUtils.test.ts

```diff
@@ -236,6 +239,117 @@ describe('AzureFunctionUtils', function (): void {
 		});
 	});
 
+	describe('Get Azure Function Files', function (): void {
+		it('Should return undefined if not azure function projects are found', async () => {
+			// set workspace folder for testing
+			sinon.replaceGetter(vscode.workspace, 'workspaceFolders', () => {
+				return <vscode.WorkspaceFolder[]>[{
+					uri: {
+						fsPath: '/temp/'
+					},
+				}];
+			});
+			let result = await azureFunctionsUtils.getAzureFunctionProject();
+			should(result).be.undefined();
+		});
+
+		it('Should return selectedProjectFile if only one azure function project is found', async () => {
+			// set workspace folder for testing
+			sinon.replaceGetter(vscode.workspace, 'workspaceFolders', () => {
+				return <vscode.WorkspaceFolder[]>[{
+					uri: {
+						fsPath: '/temp/'
+					},
+				}];
+			});
+			// only one azure function project found - hostFiles and csproj files stubs
+			let findFilesStub = sinon.stub(vscode.workspace, 'findFiles');
+			findFilesStub.onFirstCall().returns(Promise.resolve([vscode.Uri.file('/temp/host.json')]) as any);
```

Comment:
You can shortcut these to remove the Promise.resolve - e.g. `onFirst().resolves([vscode.Uri.file(...)]);`